plugins/eslint-plugin-liferay/lib/rules/group-imports.js

```diff
@@ -108,8 +108,22 @@ module.exports = {
 						argument.type === 'Literal' &&
 						typeof argument.value === 'string'
 					) {
-						if (node.parent.type === 'ExpressionStatement') {
+						if (
+							node.parent.type === 'CallExpression' &&
+							node.parent.parent.type === 'VariableDeclarator' &&
+							node.parent.parent.parent.type ===
+								'VariableDeclaration'
+						) {
+							register(node.parent.parent.parent);
+						} else if (node.parent.type === 'ExpressionStatement') {
 							register(node.parent);
+						} else if (
+							node.parent.type === 'MemberExpression' &&
+							node.parent.parent.type === 'VariableDeclarator' &&
+							node.parent.parent.parent.type ===
+								'VariableDeclaration'
+						) {
+							register(node.parent.parent.parent);
```

Comment:
So this whole approach is super manual and special-casey, and note that any such code that I put in here technically should go in `sort-imports` and maybe `imports-first` as well. Will consider ways to do this better tomorrow.